app/main.py

```diff
@@ -2,0 +1,17 @@
+from .errors import (NotWearingMaskError, VaccineError)
+
+
+def go_to_cafe(friends: list, cafe: str) -> None:
```

Comment:
The function `go_to_cafe` should return a string, but the type hint currently indicates it returns `None`. Update the type hint to `-> str` to reflect the actual return type.